ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionViewModel.kt

```diff
@@ -97,8 +98,15 @@ internal constructor(
   private var shouldLoadFromDraft: Boolean = savedStateHandle[TASK_SHOULD_LOAD_FROM_DRAFT] ?: false
   private var draftDeltas: List<ValueDelta>? = null
 
-  private val activeSurvey: Survey = requireNotNull(surveyRepository.activeSurvey)
-  private val job: Job = activeSurvey.getJob(jobId) ?: error("couldn't retrieve job for $jobId")
+  private val _activeSurvey = MutableStateFlow<Survey?>(null)
+  val activeSurvey = _activeSurvey.asStateFlow()
+
+  init {
+    viewModelScope.launch { surveyRepository.activeSurveyFlow.collect { _activeSurvey.value = it } }
```

Comment:
Isn't `activeSurvey` as you're defining it here functionally equivalent to `surveyRepository.activeSurveyFlow`? If so, why not just expose that instead?